src/pages/admin/appearance.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { AppContainer } from '@modules/admin/components';
```

Comment:
Same thing. Try to follow the `imports` pattern that we have in other components. (_I hope I've been managing them consistently tho_)

Reply:
Fixed: imports of hooks and components are separated.